src/mpyl/cli/build.py

```diff
@@ -270,6 +280,38 @@ def by_choice(choice: Choice):
         return release.tag_name
 
 
+def get_test_releases():
+    console = Console()
+    console.log("Fetching MPyL releases..")
+    url = "https://test.pypi.org/pypi/mpyl/json"
+    data = requests.get(url, timeout=30).json()
+    versions = list(data["releases"].keys())
+    versions.sort(key=LooseVersion, reverse=True)
+    versions = [version[:-4] for version in versions[:100]]
+    versions = list(dict.fromkeys(versions))
+    versions = [version + "*" for version in versions]
+    return versions
```

Comment:
You could unittest this logic by capturing a json response.

Reply:
added a unittest